tutor/resources/styles/book-content/note.less

```diff
@@ -162,7 +162,7 @@
   }
 
   // .example, check-understanding content currently lacks a data-label attribute
-  > .example,
+  .book-content > .example,
   section > .example {
     .tutor-book-before-manual-label("Worked Example");
   }
```

Comment:
hm, would it make more sense to import all the book content styles inside a `.book-content` scope?  Or would that be too large a change do you think?